setup_and_configuration/post-installation-configuration.md

```diff
@@ -2,9 +2,19 @@
 
 ### **Change Heath Center Name**
 
-1. Login and at  the home page select System Administration as shown below![](/assets/healthcentre.png)
+1.Login and at the home page select System Administration as shown below
```

Comment:
@ODORA0 the images on this page are broken and do not display

Reply:
@ssmusoke I wonder how and why that is, am basing on markdown preview on my IDE https://snipboard.io/LcCd9Y.jpg
Can't be an issue with my markdown code

Reply:
> @ssmusoke relative link like this `![Health Centre Name Display](health_centre_name_displayed.png)` rather than `![Health Centre Name Display](/assets/health_centre_name_displayed.png)` doesn't render any images on my IDE. Intellij and VS code inclusive

Nope it has to be relative to the page where the image is referenced!!

Reply:
@ssmusoke I think I don't get the relative links you might be meaning mean, becasue this is a relative link `[a relative link](other_file.md)` also for an image: `[a relative link](/img/other_file.png)`

Reply:
So I tried to open `![Health Centre Name Display](/assets/health_centre_name_displayed.png)` on VS code, it works. Intellij doesn't actually render the images or Typora for me